examples/datastores.py

```diff
@@ -88,6 +88,11 @@
 print(f"{datastore}")
 print(f"{pp.pformat(datastore.data)} \n")
 
+# Resize the datastore to 1 GB
```

Comment:
recommend getting the current size of the datastore and adding 1 GB to it instead.

Reply:
Updated the code